keps/draft-20190418-skeleton-generator.md

```diff
@@ -0,0 +1,39 @@
+---
+kep-number: draft-20190418
```

Comment:
As of our todays Community meeting this code the kep number `0007` assigned. congrats mr. bond :)

Let's not forget to change the filename, as well as kep-number here.